src/notebook/Cell.jl

```diff
@@ -40,6 +40,7 @@ Base.@kwdef mutable struct Cell
 
     code::String=""
     code_folded::Bool=false
+    isolated::Bool=false
```

Comment:
This is something I did comment about recently here:
https://github.com/fonsp/Pluto.jl/pull/2239#issuecomment-1219630198

It would be nice to have a Set of option/statuses instead of the huge amount of flags we currently have (and we will have to add as new functionality is added like in this PR)